### ISSUE TITLE
bug: langgraph JS/TS input empty when streaming

### ISSUE THREAD

**Issue by marcklingen** [langfuse/langfuse#3138]
### Describe the bug

I have code like this:
`graph.stream('some message', { callbacks: [//the usual langfuse callback] })`

And I received trace like:
![image](https://github.com/user-attachments/assets/3a08b84d-0713-4780-a10a-39d9fc819693)

### To reproduce

see above

### SDK and container versions

to be added to issue

### Are you interested to contribute a fix for this bug?

No

**Comment by HenryHengZJ** [comment]
using the latest langfuse-langchain and langfuse sdk [3.19.0](https://www.npmjs.com/package/langfuse-langchain/v/3.19.0)

**Comment by richlaiq** [comment]
I do not see this issue with langgraph  0.0.59  and langfuse  2.35.0 when I use invoke as I see a trace with an input and output. But I do not see any traces at all when using astream or stream. 

Also I am curious how 

```
graph.stream('some message', { callbacks: [//the usual langfuse callback] })
```
should work. It should be in format:

```
graph.stream({"messages':[('user', some message',)] { callbacks: [//the usual langfuse callback] })
```

In addition, Flowise has an issue similar to one on top where the input is a empty string.

**Comment by marcklingen** [comment]
thanks for the additional context @richlaiq! 

> Also I am curious how
> `graph.stream('some message', { callbacks: [//the usual langfuse callback] })`
> should work.

this was copied from the report regarding flowise, i think we can skip over this difference as this was probably just a typo when quickly reporting this issue.

flowise seems to run on langgraph `^0.0.22` according to the package lock which might contribute to this issue, thanks for confirming that you are having the same issue on `0.0.59`

**Comment by HenryHengZJ** [comment]
yeah actually @richlaiq reported to me, and I reported to you @marcklingen 😆

**Comment by marcklingen** [comment]
And we are all back together in this issue 🙂 <br><br>thanks again for reporting, we will try to have a look soon to reproduce (latest langgraph and langfuse versions) and fix this.

**Comment by richlaiq** [comment]
> I do not see this issue with langgraph 0.0.59 and langfuse 2.35.0 when I use invoke as I see a trace with an input and output. But I do not see any traces at all when using astream or stream.
> 
> Also I am curious how
> 
> ```
> graph.stream('some message', { callbacks: [//the usual langfuse callback] })
> ```
> 
> should work. It should be in format:
> 
> ```
> graph.stream({"messages':[('user', some message',)] { callbacks: [//the usual langfuse callback] })
> ```
> 
> In addition, Flowise has an issue similar to one on top where the input is a empty string.

My bad. I forgot to put the callbacks in the astream and stream versions in langraph so I see the traces for langgraph for invoke, stream and astream

**Comment by marcklingen** [comment]
@richlaiq do they also contain the input?

**Comment by richlaiq** [comment]
@marcklingen , yes for langgraph  I see input and output for invoke, stream and astream. I just do not see the input for Flowise using the components in AgentFlows.

**Comment by marcklingen** [comment]
thanks for confirming

@HenryHengZJ do agentflows in flowise use langgraph TS exclusively?

**Comment by HenryHengZJ** [comment]
@richlaiq are you testing on langgraph py? the function above is correct from JS/TS implementation (https://langchain-ai.github.io/langgraphjs/how-tos/stream-values/#stream-values)

yes, we are using langgraph ts

**Comment by richlaiq** [comment]
> @richlaiq are you testing on langgraph py? the function above is correct from JS/TS implementation (https://langchain-ai.github.io/langgraphjs/how-tos/stream-values/#stream-values)
> 
> yes, we are using langgraph ts

Hi @HenryHengZJ , I only ran some tests on langgraph py to see if has a similar issue because I was curious if this was an issue with langgraph js but not in py. The python implementation seems to be fine.

**Comment by marcklingen** [comment]
> @marcklingen , yes for langgraph I see input and output for invoke, stream and astream. I just do not see the input for Flowise using the components in AgentFlows.

did you use the TS version here? if yes, maybe flowise run on an older version as per package-lock which might cause this issue (just a hypothesis as you weren't able to reproduce this)

**Comment by marcklingen** [comment]
thanks for the clarification, I thought you tested on JS/TS, updating the issue title again as we'll need to check whether this is an issue on the latest version, can't get to this right now unfortunately

**Comment by hassiebp** [comment]
Hi all, this issue is no longer reproducible for me using the latest JS SDK (v3.25.2) and @langchain/core@0.3.3 and @langchain/langgraph@0.2.8. 

Here is my sample code that ran well: 

```typescript
import { TavilySearchResults } from "@langchain/community/tools/tavily_search";
import { ChatOpenAI } from "@langchain/openai";
import { MemorySaver } from "@langchain/langgraph";
import { HumanMessage } from "@langchain/core/messages";
import { createReactAgent } from "@langchain/langgraph/prebuilt";
import { CallbackHandler } from "./langfuse-langchain/lib/index.mjs";

const main = async (): Promise<any> => {
  // Define the tools for the agent to use
  const agentTools = [new TavilySearchResults({ maxResults: 3 })];
  const agentModel = new ChatOpenAI({ temperature: 0 });
  const handler = new CallbackHandler();

  // Initialize memory to persist state between graph runs
  const agentCheckpointer = new MemorySaver();
  const agent = createReactAgent({
    llm: agentModel,
    tools: agentTools as any,
    checkpointSaver: agentCheckpointer,
  });

  // Now it's time to use!
  const agentFinalState = await agent.invoke(
    { messages: [new HumanMessage("what is the current weather in sf")] },
    { configurable: { thread_id: "42" } }
  );

  console.log(agentFinalState.messages[agentFinalState.messages.length - 1].content);

  const stream = await agent.stream(
    { messages: [new HumanMessage("what about ny")] },
    { configurable: { thread_id: "42" }, callbacks: [handler] }
  );

  for await (const chunk of stream) {
    console.clear();
    console.log(chunk);
  }

  await handler.flushAsync();
};

console.log(main().then(() => console.log("done")));
```

Link to resulting trace: https://cloud.langfuse.com/project/cm1geuvsr027w7nuhw8g69xtb/traces/5bb23833-b1aa-4d73-8a4f-539476d11c92

Could you please upgrade and try again?

**Comment by hassiebp** [comment]
Closing this as stale, please reopen if issue persists.
